src/Microsoft.AspNetCore.Razor.OmniSharpPlugin/FallbackConfigurationProvider.cs

```diff
@@ -29,13 +29,8 @@ public override bool TryResolveConfiguration(ProjectConfigurationProviderContext
             foreach (var compilationReference in compilationReferences)
             {
                 var assemblyFullPath = compilationReference.EvaluatedInclude;
-                if (assemblyFullPath.EndsWith(MvcAssemblyFileName, StringComparison.OrdinalIgnoreCase))
+                if (assemblyFullPath.EndsWith(MvcAssemblyFileName, FilePathComparison.Instance))
                 {
-                    if (assemblyFullPath.Length == MvcAssemblyFileName.Length)
-                    {
-                        continue;
-                    }
-
                     var potentialPathSeparator = assemblyFullPath[assemblyFullPath.Length - MvcAssemblyFileName.Length - 1];
```

Comment:
Is there a chance `assemblyFullPath.Length - MvcAssemblyFileName.Length - 1` will end up a negative value?

Reply:
Hmm, only if the include happens to exactly be `MvcAssemblyFileName` (don't think MSBuild allows that). I can protect against it either way, no harm.

Reply:
I'm just being paranoid. If it's not practical don't bother.